### PR TITLE
Hemophage Update (Nerfs and Changes)

### PR DESCRIPTION
**Pull request by Vizoplez**
## About The Pull Request
Makes Hemophage less susceptible to damage while Dormant, as the debuffs while already dormant are pretty massive (damage multiplier + no regen)

## How This Contributes To The Skyrat Roleplay Experience
This makes them less of an 'instant pick' race for things like 'Vampire Moths', and generally broadens the idea of Hemophages to include other species that have digi legs.

## Proof of Testing
Webedit L

<details>
<summary>Screenshots/Videos</summary>
  
</details>

## Changelog

:cl:
balance: Hemophage blood drain made more lethal for space law/assault reasons
balance: Hemophage blood loss buffed slightly
balance: Hemophage damage multiplier when dormant reduced
balance: Hemophages now have a VERY bad time without their tumor, and will basically be RR'd without it.
tweak: Hemophages now use mutant parts + mutant body overrides.
/:cl:

### CODE REVIEW

**Comment by ReturnToZender** [comment]
> TLDR: This makes Hemophages far less snowflakey, and actually criminal if they actively feed on people. Currently (before changes) You could feed on someone ~3x before putting them into unconciousness from bloodloss, requiring 2 more feeds to guaranteedly kill them.
> 
> Now you kill them in ~3 feeds, 1 makes them woozy to the point where they'll pass out after ~60 seconds, 2 instantly drops them, and 3 puts them to lethal levels.
> 
> Is this 'fun' for the Hemophage? Fuck no. If you want to play a snowflake race that passively regens ~1 brute/burn a tick and sucks blood, expect to be criminalised if you feed on sapient beings without their consent.
> 
> Again. Plasmamen are still charged with assault if they take their helmet off to catch fire and run into people. This should be no different.
> 
> Failure to compile is due to what seems to be a residual screenshot check.

This looks like a good concept to me. The danger is part of the appeal, and the timer is pretty generous. I think hemophages will have to properly work to earn that blood.

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm [hunk 1]
```diff
@@ -1,15 +1,15 @@
 /// Maximum an Hemophage will drain, they will drain less if they hit their cap.
-#define HEMOPHAGE_DRAIN_AMOUNT 50
+#define HEMOPHAGE_DRAIN_AMOUNT 150 //Hemophages are now more lethal when they drain living people. This changes the charge of unwilling feeding to assault by proxy, instead of battery - Bubberstation change
```
Comment (Cyprex) [inline modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm:2]: People might use this for RP or other such things, why nerf them so significantly by increasing this to immediately dangerous levels?

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 /// How much blood do Hemophages normally lose per second (visible effect is every two seconds, so twice this value).
-#define NORMAL_BLOOD_DRAIN 0.125
+#define NORMAL_BLOOD_DRAIN 0.150 //.025 increase to make them need more blood.
 /// Minimum amount of blood that you can reach via blood regeneration, regeneration will stop below this.
 #define MINIMUM_VOLUME_FOR_REGEN (BLOOD_VOLUME_BAD + 1) // We do this to avoid any jankiness, and because we want to ensure that they don't fall into a state where they're constantly passing out in a locker.
 /// Minimum amount of light for Hemophages to be considered in pure darkness, and therefore be allowed to heal just like in a closet.
 #define MINIMUM_LIGHT_THRESHOLD_FOR_REGEN 0
 /// How much organ damage do all hemophage organs take per second when the tumor is removed?
-#define TUMORLESS_ORGAN_DAMAGE 5
+#define TUMORLESS_ORGAN_DAMAGE 10
```
Comment (Cyprex) [inline modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm:10]: What's a tumor?

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm [hunk 3]
```diff
@@ -69,24 +69,43 @@
...
+	bodypart_overrides = list(
+		BODY_ZONE_HEAD = /obj/item/bodypart/head/mutant,
+		BODY_ZONE_CHEST = /obj/item/bodypart/chest/mutant,
+		BODY_ZONE_L_ARM = /obj/item/bodypart/arm/left/mutant,
+		BODY_ZONE_R_ARM = /obj/item/bodypart/arm/right/mutant,
+		BODY_ZONE_L_LEG = /obj/item/bodypart/leg/left/mutant,
+		BODY_ZONE_R_LEG = /obj/item/bodypart/leg/right/mutant,
+	) //Bubberstation change. Allows them to use the mutant bodyparts of other species like anthromorph.
+
+	veteran_only = FALSE
```
Comment (Cyprex) [inline modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm:108]:
```suggestion
```

Comment (Cyprex) [reply modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm:108]: I'd rather not edit these needlessly

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm (hunk 1 above)
Comment (Cyprex) [reply modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm:2]: Also, can this be used offensively as a weapon?

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm [hunk 4]
```diff
@@ -69,24 +69,43 @@
...
+		"legs" = "None",
+		"taur" = "None",
+		"fluff" = "None",
+		"wings" = "None",
+		"head_acc" = "None",
+		"neck_acc" = "None"
+	)
 	exotic_bloodtype = "U"
-	use_skintones = TRUE
+	use_skintones = FALSE
```
Comment (Cyprex) [inline modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm:86]: What does this actually do

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm [hunk 5]
```diff
@@ -69,24 +69,43 @@
...
 	var/tumor_status = PULSATING_TUMOR_MISSING
-
-	veteran_only = TRUE
+	bodypart_overrides = list(
+		BODY_ZONE_HEAD = /obj/item/bodypart/head/mutant,
+		BODY_ZONE_CHEST = /obj/item/bodypart/chest/mutant,
+		BODY_ZONE_L_ARM = /obj/item/bodypart/arm/left/mutant,
+		BODY_ZONE_R_ARM = /obj/item/bodypart/arm/right/mutant,
+		BODY_ZONE_L_LEG = /obj/item/bodypart/leg/left/mutant,
+		BODY_ZONE_R_LEG = /obj/item/bodypart/leg/right/mutant,
```
Comment (Cyprex) [inline modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm:105]: Do these & the default_mutant_bodyparts change affect existing characters in any way?

**Review by Cyprex — CHANGES_REQUESTED** [review]
